res/database/Default/database_news.xml

```diff
@@ -8954,14 +8790,18 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
 			<title>
 				<de>Öl in ${leipzig} gefunden</de>
 				<en>Oil found in ${leipzig}</en>
+				<pl>${leipzig}: Znaleziono olej</pl>
 			</title>
 			<description>
 				<de>Bei Baugrunduntersuchungen in ${leipzig} haben Geologen wider Erwarten Öl gefunden.</de>
 				<en>Contrary to expectations, geologists have found oil during subsoil investigations in ${leipzig}.</en>
+				<pl>${leipzig}: Wbrew oczekiwaniom geolodzy znaleźli ropę naftową podczas badań podglebia</pl>
 			</description>
 			<variables>
 				<leipzig>
-					<all>Leipzig|Dresden|Halle|Erfurt|Magdeburg|Potsdam|Rostock|Chemnitz|Cottbus</all>
+					<de>Leipzig|Dresden|Halle|Erfurt|Magdeburg|Potsdam|Rostock|Chemnitz|Cottbus</de>
```

Comment:
you can still use all, but only override one language

Reply:
yes though as in some other instances more of a stylistic choice when the names are the same more or less by coincidence.